Messaging/src/RabbitMQWeb/Controllers/RabbitController.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using RabbitMQWeb.Models;
+using Steeltoe.Messaging.RabbitMQ.Core;
+
+namespace RabbitMQWeb.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class RabbitController : ControllerBase
+    {
+        private readonly ILogger<RabbitController> _logger;
+        private readonly RabbitTemplate _rabbitTemplate;
+        private readonly RabbitAdmin _rabbitAdmin;
+
+        public RabbitController(ILogger<RabbitController> logger, RabbitTemplate rabbitTemplate,
+            RabbitAdmin rabbitAdmin)
+        {
+            _logger = logger;
+            _rabbitTemplate = rabbitTemplate;
+            _rabbitAdmin = rabbitAdmin;
+        }
+
+        [HttpGet()]
+        public ActionResult<string> Index()
+        {
+            return @"
+You can use these endpoints to interact with RabbitMQ
+    /Rabbit/SendRabbitMessage
+    /Rabbit/SendLongEaredRabbitMessage
+    /Rabbit/SendReceiveRabbitMessage
+    /Rabbit/SendReceiveLongEaredRabbitMessage
+    /Rabbit/DeleteQueues
+";
```

Comment:
```suggestion
        You can use these endpoints to interact with RabbitMQ
            /Rabbit/SendRabbitMessage
            /Rabbit/SendLongEaredRabbitMessage
            /Rabbit/SendReceiveRabbitMessage
            /Rabbit/SendReceiveLongEaredRabbitMessage
            /Rabbit/DeleteQueues
        ";
```

Reply:
Just adding some spacing for alignment. This may not be the proper amount of space to align inside the return, so feel free to adjust as needed.

Reply:
This is the body that gets returned so it's what the end user gets in response.  @"'s are a funky beast when it comes to formatting.  The content needs to be left justified the the 0 column.  But I never am quite sure what to do with starting ".  I think the code here should be as committed in PR.